Add new routes for administration pages

FAEZ10 opened the pull request:
# Pull Request Templates

## Type of change

- [ ] Feature
- [ ] Fix

## Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

FAEZ14 committed:
fix(size) add required and default option in model

FAEZ14 committed:
Feat add new routes

FAEZ14 committed:
Merge branch 'dev' into feature/improve-api-route

FAEZ14 committed:
Remove logs

johnnyhelloworld commented on src/apps/auth/domain/user/user.service.ts:
```diff
@@ -89,6 +88,7 @@ export class UserService extends BaseService {
         }
         return matchedRole;
       });
+      console.log("validRoles", validRoles);
```
remove console.log()

FAEZ14 committed:
feat(order): add admin route to create orders for users

- Implement POST /orders/admin/users/:userId/order endpoint
- Update AdminCreateOrderDTO to accept only productId and quantity
- Modify OrderService.createOrderForUser to handle new payload structure
- Add validation for incoming order data without price fields
- Ensure proper error handling and response messages
- Update OrderController to include the new route

FAEZ14 committed:
Remove logs
